packages/analytics-js/src/services/ErrorHandler/ErrorHandler.ts

```diff
@@ -109,12 +85,7 @@ class ErrorHandler implements IErrorHandler {
     if (!errorMessage) {
```

Comment:
This will filter the PromiseRejectionEvent type errors. So, we should have a separate normalization strategy for handled and unhandled errors.